localserver.command

```diff
@@ -1,3 +1,12 @@
-cd `dirname $0`
+#!/bin/bash
+
+cd "$(dirname "$0")"
 open http://0.0.0.0:8000/
-python -m SimpleHTTPServer
+
+# Attempt to start the server using SimpleHTTPServer
+if python -m SimpleHTTPServer &>/dev/null; then
+  pass
+else
+  # Fallback to Python 3's http.server
+  python3 -m http.server 8000
```

Comment:
change needed for newer macs which don't have `python`, but `python3` installed by default